logistic_requisition/model/logistic_requisition.py

```diff
@@ -981,6 +1029,17 @@ def _check_sourcing_procurement(self):
         return []
 
     @api.multi
+    def _check_sourcing_reuse_bid(self):
+        """Check sourcing for "reuse_bid" method.
+
+        :returns: list of error strings
+
+        """
+        if not self.selected_bid_line_id:
+            return ['{0}: No bid line selected'.format(self.name)]
```

Comment:
Is it normal that it is not translated? (same question for the other `_check_sourcing*' methods)